jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/PersistenceUnitParseProcessor.java

```diff
@@ -283,6 +283,8 @@ private void postParseSteps(
             }
 
             pu.setScopedPersistenceUnitName(scopedPersistenceUnitName);
+
+            pu.setContainingModuleName(getContainingModuleName(deploymentUnit));
```

Comment:
Here we make note of the actual deployment module name containing the persistence unit (e.g. ear name + jar/war sub-deployment name).